dashboard/app/views/levels/editors/fields/_display_area.html.haml

```diff
@@ -0,0 +1,10 @@
+%legend.control-legend.collapsed{data: {toggle: "collapse", target: "#display-area"}}
+  Display Area
+#display-area.in.collapse
+  - if @level.is_a?(Applab)
+    = render partial: 'levels/editors/fields/applab_design_mode', locals: {f: f}
+    = render partial: 'levels/editors/fields/applab_data', locals: {f: f}
```

Comment:
again I'm confused about the inclusion of app mode tabs as part of the "display area". perhaps this section could be for "app mode tabs" and leav out the control buttons?

Reply:
I ungrouped these fields. Thanks for the feedback